### ISSUE TITLE
Before starting an analysis, you should be able to combine metadata categories

### ISSUE THREAD

**Issue by ElDeveloper** [biocore/qiita#1394]
An interface component should be provided such that categories can be freely combined prior the generation of any of the visualizations. Users should be able to for example select `BODY_SITE` and `STUDY_TITLE` and create a new category called `BODY_SITE_AND_STUDY_TITLE`

**Comment by antgonza** [comment]
This is a duplicate of: https://github.com/biocore/qiita/issues/1446
